contents/docs/data/sessions.mdx

```diff
@@ -12,50 +12,80 @@ export const uniqueSessionLight = "https://res.cloudinary.com/dmukukwp6/image/up
 export const uniqueSessionDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/unique-sessions-dark-mode.png"
 export const sessionFormulaLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/session-formula-light-mode.png"
 export const sessionFormulaDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/session-formula-dark-mode.png"
-export const durationFilterLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/duration-filter-light-mode.png"
-export const durationFilterDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/duration-filter-dark-mode.png"
 export const medianDurationLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/median-duration-light-mode.png"
 export const medianDurationDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/median-duration-dark-mode.png"
-export const durationBreakdownLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/duration-breakdown-light-mode.png"
-export const durationBreakdownDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/sessions/duration-breakdown-dark-mode.png"
 
-## What is a session?
+A session is a set of events that try to capture a single use of your product or visit to your website. They are used in multiple areas of PostHog:
```

Comment:
there is a weirdness that session duration exposes (and we should maybe measure to see whether we should care)

the event span of a session is not necessarily the same as the recording span of a session 

and i worry that the type of users we have will spot this 

so for e.g. on first visit you probably have a page view event while you're downloading the lazyily loaded replay JS bundle and then recording starts so that session start is offset from the recording start.

then if you don't have pageleave enabled your last event in a session could be much earlier than the last recording activity

so if we let people query for recordings based on session duration they'll see recordings with maybe wildly different duration values

---

as i type this is makes sense to me to say that a session is canonically the event span 

when I'm back from leave (unless someone is eager to do this) I'll look at what the difference between session duration and session recording duration is. then we can decide if we need other content or in-app explanations

Reply:
I see what you mean, there is [this question](https://posthog.com/questions/simple-question) asking about inactivity and session duration which I think this isn't super clear about.

As long as we are clear what session duration is defined as and why replay might be different, I think we are good.